### PR TITLE
Custom Tab Implementation

### PR DESCRIPTION
**Pull request by avidraghavMM**
Custom Tab Implementation as an efficient and more customisable option against WebView.

### CODE REVIEW

File: app/src/main/java/com/raghav/spacedawn/ui/MainActivity.kt [hunk 1]
```diff
@@ -42,29 +44,25 @@ class MainActivity : AppCompatActivity() {
...
                         )
                     } catch (e: ActivityNotFoundException) {
-                        startActivity(
-                            Intent(
-                                ACTION_VIEW,
-                                Uri.parse("http://play.google.com/store/apps/details?id=$packageName")
-                            )
+                        customTabIntent.launchUrl(
+                            this,
+                            Uri.parse("http://play.google.com/store/apps/details?id=$packageName")
```
Comment (aditya-bhawsar-mm) [inline app/src/main/java/com/raghav/spacedawn/ui/MainActivity.kt:49]: in this case, we can rely on ACTION_VIEW intents here to launch the play store before this ?

**Review by aditya-bhawsar-mm — COMMENTED** [review]
Please consider adding checks for edge case of no application being present to handle custom tabs

**Review by aditya-bhawsar-mm — APPROVED** [review]
Try to address the comments

File: app/src/main/java/com/raghav/spacedawn/ui/MainActivity.kt (hunk 1 above)
Comment (avidraghavMM) [reply app/src/main/java/com/raghav/spacedawn/ui/MainActivity.kt:49]: I read this from the documentation hence didn't added any check
https://developer.chrome.com/docs/android/custom-tabs/integration-guide/#what-happens-if-the-user-doesnt-have-a-browser-that-supports-custom-tabs-installed
Screenshot 2023-02-27 at 7 54 34 PM

**Pull request merged**